language/move-lang/src/expansion/translate.rs

```diff
@@ -273,6 +288,10 @@ fn main(
             *loc,
             "Extraneous 'public' modifier. This top-level function is assumed to be public",
         )]),
+        FunctionVisibility::Spec(loc) => context.error(vec![(
+            *loc,
+            "Extraneous 'spec' modifier. This top-level function is assumed to be public",
```

Comment:
Probably want to say something like. "Extraneous 'spec' modifier. Move prover specification functions can only be declared inside modules"

Reply:
Done